jargons/grunt.md

```diff
@@ -0,0 +1,17 @@
+---
+title: grunt
+tags:
+- javascript
+---
+
+En basit anlatımıyla; [Node.js](/nodejs) ile javascript üzerindeki bağımlılıklarınızı yöneten, derleyen ve birleştiren araçtır, bunun yanı sıra eklentileri ile birlikte pek çok görev otomasyonu için kullanılabilir.
+
+Örnek Görevler; 
+*  Html, Javascript, ECMAScript, Css ve Görüntü dosyalarını minimize ve derleme işlemleri.
+*  Sass, Less dosyalarının Css’e derlenmesi ve minimize edilmesi.
+*  Css sprite oluşturma.
+
+Grunt eklentilerini [buradan](https://gruntjs.com/plugins) inceleyebilirsiniz.
+
+Alternatifler: [Webpack](/webpack), [Gulp](/gulp)
```

Comment:
Bence de grunt/gulp ikilisi ile webpack'in cozmeye calistiklari seyler birebir ayni degil o yuzden buradan webpack'i kaldirsak daha dogru olacak gibi.

Reply:
👍